ios/Sources/Model/Media.swift

```diff
@@ -0,0 +1,24 @@
+import DroidKaigiMPP
+
+public enum Media: Equatable {
+    case droidkaigifm(String)
+    case medium(String)
+    case youtube(String)
+    case other(String)
+
+    static func from(_ model: DroidKaigiMPP.Media) -> Media {
+        switch model {
+        case let fm as DroidKaigiMPP.Media.DroidKaigiFM:
+            return .droidkaigifm(fm.text)
+        case let medium as DroidKaigiMPP.Media.Medium:
+            return .medium(medium.text)
+        case let youtube as DroidKaigiMPP.Media.YouTube:
+            return .youtube(youtube.text)
+        case let other as DroidKaigiMPP.Media.Other:
+            return .other(other.text)
```

Comment:
![badge](https://img.shields.io/badge/review-ask-blue.svg)
Is the `text` necessary for us?
It seems like the raw value for saving to the database.

Reply:
> It seems like the raw value for saving to the database.

I see. I didn't understand about it 😢 
I will remove it 🙏🏼 
If there is any inconvenience later, I will revert it.